### DISCUSSION TITLE
Is it possible to empty the player’s inventory?

### DISCUSSION THREAD

**Discussion by BuzzLEquerre** [flareteam/flare-game#982] [Q&A] [answered]
hello I would like to make an arena for my game where the player can go back taking the equipment offered only I do not want the player to keep his classic inventory once in the arena.
I would like when the player enters the arena his inventory is emptied in a safe at the entrance.
Is this feasible ?

**Comment by dorkster** [comment, marked as answer]
There are a few things that would prevent you from implementing that as you describe:
1. The "safe" would need to be the player's stash, as that's the only persistent item container available.
2. There's no implemented way to automatically transfer things to the player's stash.

I think you could get close, though. You could prompt the player to remove their gear themselves. The event that allows entry to the arena could use `requires_not_item` for all pieces of equipment. Although, that approach won't work if you use generated items, since you can't know the IDs ahead of time. Also, it might be kind of tedious if you have a lot of items.

**Reply by BuzzLEquerre** [reply]
so if I proceed with an entrance room that requires to have 0 items to enter, then I define a "no_item" status and then the player chooses via an NPC his fighter class for the arena. And once chosen, this distributes a status to the "arena_ready" player that allows him to enter the arena. With this method I could make my arena as a result, thanks to you for giving me the command "requires_not_item" that I didn't know.

I'll try to make a prototype this week I'll share it with you :)
